ipwb/replay.py

```diff
@@ -787,12 +785,17 @@ def generateNoMementosInterface_noDatetime(urir):
     return msg
 
 
+@app.errorhandler(404)
+def page_not_found(e):
+    return "<h1>ERROR 404</h1>Not Found", 404
+
+
 def generateNoMementosInterface(path, datetime):
     msg = '<h1>ERROR 404</h1>'
-    msg += 'No capture found for {0} at {1}.'.format(path, datetime)
+    msg += f'No capture found for {path} at {datetime}.'
```

Comment:
* Wrapped in paragraph tag
* Used plural form of `captures`

```suggestion
    msg += f'<p>No captures found for {path} at {datetime}.</p>'
```